modules/passkey/test/SafeWebAuthnSignerFactory.spec.ts

```diff
@@ -0,0 +1,178 @@
+import { expect } from 'chai'
+import { deployments, ethers } from 'hardhat'
+import { DUMMY_AUTHENTICATOR_DATA, base64UrlEncode, encodeWebAuthnSigningMessage, getSignatureBytes } from './utils/webauthn'
+
+describe('SafeWebAuthnSignerFactory', () => {
+  const setupTests = deployments.createFixture(async () => {
+    const { SafeWebAuthnSignerFactory } = await deployments.fixture()
+    const factory = await ethers.getContractAt('SafeWebAuthnSignerFactory', SafeWebAuthnSignerFactory.address)
+
+    const MockContract = await ethers.getContractFactory('MockContract')
+    const mockVerifier = await MockContract.deploy()
+
+    return { factory, mockVerifier }
+  })
+
+  describe('getSigner', function () {
+    it('Should return the address that a signer will be created on', async () => {
```

Comment:
The test description feels a bit ambiguous based on the `expect` we check in this test.

Reply:
Hmm, to me it makes sense. We are checking that the address returned to by `getSigner` is the address where the signer will be deployed to when creating it.

So we get the address, deploy the signer, and check there is code there, i.e.:
> [`getSigner` ...] returns the address that a signer will be created on